storage/api/Storage.Samples/RemoveBucketIamMember.cs

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START storage_remove_bucket_iam_member]
+
+using Google.Cloud.Storage.V1;
+using System;
+using System.Linq;
+
+public class RemoveBucketIamMemberSample
+{
+    public void RemoveBucketIamMember(string bucketName, string role, string member)
```

Comment:
See the comment I left in the add iam member sample about the role and member variables